src/pages/Login/LoginPage.tsx

```diff
@@ -79,7 +79,16 @@ export default class LoginPage extends React.Component<LoginProps, LoginState> {
                 <Col sm={10} smOffset={1} md={8} mdOffset={2} lg={8} lgOffset={2}>
                   <div className={'card-pf'}>
                     <header className={'login-pf-header'} />
-                    {this.props.status === LoginStatus.error && <Alert>{this.props.message}</Alert>}
+                    {authenticationConfig.secretMissing && (
+                      <Alert>
+                        The Kiali secret is missing. Users are prohibited from accessing Kiali until an administrator
+                        creates a valid secret and restarts Kiali. Please refer to the Kiali documentation for more
```

Comment:
while we are here, we can change this message slightly. We no longer need to restart Kiali. Once you create the secret, Kiali will eventually autodetect it and use the credentials it finds.

So - remove the words "and restarts Kiali" here.

```suggestion
                        creates a valid secret. Please refer to the Kiali documentation for more
```

Reply:
Agreed.
Change is applied.

Reply:
I say "we no longer need to restart Kiali" but this PR actually broke that feature and causes us to have to restart Kiali (at least that is what I am seeing). This is a regression. We need to fix that.